share/completions/nmcli.fish

```diff
@@ -57,7 +69,7 @@ complete -c nmcli -n "contains_seq connection show -- (commandline -xp)" -l orde
 complete -c nmcli -n "contains_seq connection show -- (commandline -xp)" -xa "help $cname" -k
 complete -c nmcli -n "contains_seq connection up -- (commandline -xp) && not contains -- (commandline -xp)[-1] ifname ap" -xa "help $cname" -k
 complete -c nmcli -n "contains_seq connection up -- (commandline -xp) && test (commandline -xp)[-1] = ifname" -xa "$ifname" -k
-complete -c nmcli -n "contains_seq connection up -- (commandline -xp) && test (commandline -xp)[-1] = ap" -xa "$bssid"
+complete -c nmcli -n "contains_seq connection up -- (commandline -xp) && test (commandline -xp)[-1] = ap" -xa "(__fish_nmcli_bssids)"
```

Comment:
I don't think this change has any functional effect.

The existing

	complete ... -xa "$bssd"

would expand to

	complete ... -xa '(nmcli -g BSSID device wifi list 2>/dev/null | string replace --all \\\ "")\t"BSSID"'

meaning that nmcli is only evaluated when completing "nmcli ... connection up " or similar.

I think you observed the problem that was solved in
9116c617368 (completions/nmcli: Complete at runtime, 2025-01-27).
But that commit has not been included in a release yet:

```
$ git describe --contains --tags 9116c617368
fatal: cannot describe '9116c6173686558ba873f1667e4abd325c394b04'
```

I guess we can cherry-pick it.